examples/gno.land/r/demo/microblog/microblog.gno

```diff
@@ -0,0 +1,51 @@
+// microblog is a website with shortform posts from users.
+// the API is exteremly simple - "AddPost" takes markdown and
+// adds it to the users site. The user is determined by
+// the address and your microblog lives at
```

Comment:
This should probably be reworded to:

`The microblog location is determined by the user address`

What do you think?

Reply:
Yeah, that is more clear.